server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/ProgramEndNode.java

```diff
@@ -45,21 +39,20 @@ public ProgramEndNode(Locality locality, String id, MessageService messageServic
         .map(ProgramNode.class::cast)
         .ifPresent(node -> {
           if (node.getProgramName() == null) {
```

Comment:
Please, log these errors with "debug" level

Reply:
How about logging all errors in debug mode after processing? I mean to do it generally in one place.

Reply:
The reason to do it in-place is to keep tracking where they were created, e.g. 
    `LOG.debug("Syntax error by CobolLineReaderImpl: {}", error.toString());`

Reply:
I see.